src/parser/rogue/assassination/modules/spells/EarlyDotRefresh.ts

```diff
@@ -43,7 +43,7 @@ class EarlyDotRefresh extends EarlyDotRefreshesCore {
   }
   
   // Checks the status of the last cast and marks it accordingly.
-  getLastBadCastText(event, dot) {    
+  getLastBadCastText(event: any, dot: any) {    
     if (dot.castId === SPELLS.RUPTURE.id) {
       return super.getLastBadCastText(event,dot) + " *Based on the amount of CPs spent.";
```

Comment:
Please tie these to an actual event or type and not `any`

Reply:
What type would `dot` be here? I'd assume `event` would be `CastEvent`(?) but not sure what type `dot` would be for this.